fe/fe-core/src/main/java/com/starrocks/analysis/ShowColumnStmt.java

```diff
@@ -124,17 +125,14 @@ public void init() {
     }
 
     @Override
-    public SelectStmt toSelectStmt(Analyzer analyzer) throws AnalysisException {
+    public QueryStatement toSelectStmt() throws AnalysisException {
         if (where == null) {
             return null;
         }
-        if (selectStmt != null) {
```

Comment:
selectStmt always be null now?

Reply:
Yes, no special judgment is required, and a SelectStmt is stored. Show statements are all executed once